lib/rust/ast/generation/src/types.rs

```diff
@@ -56,30 +57,30 @@ pub struct JName {
 pub fn builtin(name:&Name) -> Option<JName> {
    let name = match name.str.as_str() {
        // ERASED
-       "Box"                             => JName {jni:"",  scala:""       },
+       "Box"          => JName {jni:"",  scala:""       },
        // PRIMITIVES (https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/types.html)
-       "usize" | "isize" | "u64" | "i64" => JName {jni:"J", scala:"Long"   },
-       "u32"   | "i32"                   => JName {jni:"I", scala:"Int"    },
-       "u16"   | "i16"   | "i8"          => JName {jni:"S", scala:"Short"  },
-       "u8"                              => JName {jni:"B", scala:"Byte"   },
-       "f64"                             => JName {jni:"D", scala:"Double" },
-       "f32"                             => JName {jni:"F", scala:"Float"  },
-       "boolean"                         => JName {jni:"Z", scala:"Boolean"},
-       "char"                            => JName {jni:"C", scala:"Char"   },
-        _                                => stdlib(name)?,
+       "i64"          => JName {jni:"J", scala:"Long"   },
```

Comment:
Why did you remove the rest of these?

Reply:
Because they can contain values too big.